atlasdb-commons/src/main/java/com/palantir/common/base/BatchingVisitableView.java

```diff
@@ -186,95 +200,107 @@ public T getMax(final Ordering<? super T> o, @Nullable T defaultElement) {
      * defaultElement and have the ordering throw on null elements so you know
      * the visitable doesn't have any nulls.
      */
-    public T getMin(final Ordering<? super T> o, @Nullable T defaultElement) {
-        return BatchingVisitables.getMin(delegate(), o, defaultElement);
+    public T getMin(Ordering<? super T> ordering, @Nullable T defaultElement) {
+        return BatchingVisitables.getMin(delegate(), ordering, defaultElement);
     }
 
     /**
-     * Returns an immutable copy of the current contents of this iterable view. Does not support
-     * null elements.
+     * Returns an immutable copy of the elements in this visitable.
```

Comment:
Yea I think for this to actually address #826 it needs to be clear that this basically throws out all your paging/batching and return ALL the things.